app/models/poll/question/answer.rb

```diff
@@ -10,8 +10,26 @@ class Poll::Question::Answer < ActiveRecord::Base
   has_many :videos, class_name: 'Poll::Question::Answer::Video'
 
   validates :title, presence: true
+  validates :given_order, presence: true, uniqueness: { scope: :question_id }
+
+  before_validation :set_order, on: :create
```

Comment:
I finally used `before_validation on: :create` to set the order, because if it was set `before_create`, the validation run before all of that (so, before `set_order` method) and it failled.
For the 1st there was no problem, but for the 2nd it couldn't pass the validation and it couldn't create the record.